mockga/index.js

```diff
@@ -0,0 +1,40 @@
+/** This is purely for local development. It's a fake/mock Google Analytics
+ * server endpoint.
+ * It's not strictly working as Google Analytics does but enough to foreground
+ * log the collect POSTs coming in.
+ */
+const express = require('express');
+const chalk = require('chalk');
+
+const app = express();
+const port = parseInt(process.env.PORT || '7000');
+
+app.get('/', (req, res) => res.send('Hello World!'));
+
+app.post('/collect', (req, res) => {
+    const requiredKeys = ['v', 'tid', 'cid', 'ec', 'ea', 'el'];
+    for (let key of requiredKeys) {
+        if (!(key in req.query)) {
+            return res.status(400).send(`Missing query key '${key}'`);
+        }
+    }
+    const { ec, ea, el, cid } = req.query;
+    function showValue(s, maxLength) {
+        if (s.length > maxLength) {
+            return s.slice(0, maxLength - 1) + '…';
+        }
+        return s.padEnd(maxLength);
+    }
+    const parts = [
+        chalk.bold('ec: ') + chalk.green(showValue(ec, 20)),
+        chalk.bold('ea: ') + chalk.green(showValue(ea, 30)),
+        chalk.bold('el: ') + chalk.green(showValue(el, 50))
+    ];
+    const output = `[${new Date().toISOString()}] GA TRACK EVENT cid=${showValue(
+        cid,
+        10
+    )} | ${parts.join(' ')}`;
+    console.log(output);
```

Comment:
So for now, I'm going to venture to say it's enough to just console log print these. It looks like this, in the `docker-compose` logs (or foreground): 
```
[2020-01-07T20:25:31.197Z] GA TRACK EVENT cid=1161ff2e-… | ec: CATEGORY             ea: MYACTION                        el: MY LABEL
```

It's not perfect but it'll at least give you something. 
We could consider mounting a filesystem and have it write to a .log file that we add to `.gitignore`. Let's perhaps deal with that the day it matters.